app/controllers/courses_controller.rb

```diff
@@ -1926,7 +1926,10 @@ def copy_course
   #   - 'public_domain' (Public Domain).
   #
   # @argument course[is_public] [Boolean]
-  #   Set to true if course if public.
+  #   Set to true if course is public.
+  #
+  # @argument course[is_public_to_auth_users] [Boolean]
```

Comment:
The API response for courses already includes `is_public_to_auth_users`; I've added some doc to clarify the difference between `is_public` and `is_public_to_auth_users`.